packages/velo-external-db/lib/app.js

```diff
@@ -38,13 +49,22 @@ load().then(({ secretKey }) => {
 
     app.use('/assets', express.static(path.join(__dirname, '..', 'assets')))
     app.use(express.json())
-    app.use(unless(['/', '/provision', '/favicon.ico'], authMiddleware({ secretKey: secretKey })))
+    app.use(require('cookie-parser')())
+    app.use(session({ secret: 'secret-key', resave: false, saveUninitialized: false }))
+    app.use(passport.initialize())
+    app.use(passport.session())
+
+    app.use(unless(['/', '/provision', '/favicon.ico', '/auth/login', '/auth/callback', '/auth/logout'], secretKeyAuthMiddleware({ secretKey: secretKey })))
     config.forEach( ( { pathPrefix, roles }) => app.use(includes([pathPrefix], authRoleMiddleware({ roles }))))
+    app.all('/', authMiddleware)
```

Comment:
Used `app.all` and not `app.use`, because `app.all` will check only the path `/` and not like `app.use` that will check all the routes that includes `/`